lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -44,7 +46,10 @@
                 },
 
                 render: function () {
-                    this.$el.html(_.template(teamsTemplate));
+                    HtmlUtils.setHtml(
+                        this.$el,
+                        HtmlUtils.template(teamsTemplate)()
```

Comment:
I thought I saw it used differently above, but don't see that now.  Thanks for the explanation.